src/loggers/bt_zmq_publisher.cpp

```diff
@@ -75,6 +75,9 @@ PublisherZMQ::PublisherZMQ(const BT::Tree& tree,
             }
             catch (zmq::error_t& err)
             {
+                if (err.num() == ETERM)
```

Comment:
Not my code base to tell you what to do, but `if` without brackets makes my skin crawl. Might want to make it explicit like the styling of the rest of the file (+ L177 below)

Reply:
The python perspective argument is really good.
This burned in somehow as a nice little c++ feature.. 
But better to skip stuff like that and keep it more readable / less future error prone..

Reply:
I'm also Mr. Explicit in my code, so I take the exact opposite extreme. Do as you like here, not up to me :smile: